tests/functional/Codeception/Docker.php

```diff
@@ -0,0 +1,261 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Test\Functional\Codeception;
+
+use Codeception\Module;
+use Magento\MagentoCloud\Test\Functional\Robo\Tasks as MagentoCloudTasks;
+use Robo\LoadAllTasks as RoboTasks;
+use Codeception\TestInterface;
+use Codeception\Configuration;
+use Robo\Collection\CollectionBuilder;
+use Robo\Contract\BuilderAwareInterface;
+use League\Container\ContainerAwareInterface;
+use League\Container\ContainerAwareTrait;
+
+/**
+ * Module for running commands on Docker environment
+ */
+class Docker extends Module implements BuilderAwareInterface, ContainerAwareInterface
+{
+    use ContainerAwareTrait;
+
+    use RoboTasks, MagentoCloudTasks {
+        RoboTasks::getBuilder insteadof MagentoCloudTasks;
+        RoboTasks::setBuilder insteadof MagentoCloudTasks;
+        RoboTasks::collectionBuilder insteadof MagentoCloudTasks;
+        RoboTasks::getBuiltTask insteadof MagentoCloudTasks;
+        RoboTasks::task insteadof MagentoCloudTasks;
+    }
+
+    const BUILD_CONTAINER = 'build';
+    const DEPLOY_CONTAINER = 'deploy';
+
+    protected $config = [
+        'db_host' => '',
+        'db_port' => '3306',
+        'db_username' => '',
+        'db_password' => '',
+        'db_path' => '',
+        'repo_url' => '',
+        'repo_branch' => '',
+        'system_ece_tools_dir' => '',
+        'system_magento_dir' => '',
+        'env_base_url' => '',
+        'env_secure_base_url' => '',
+    ];
+
+    /**
+     * @inheritdoc
+     */
+    public function _initialize()
+    {
+        $container = require Configuration::projectDir() . 'tests/functional/bootstrap.php';
+        $builder = CollectionBuilder::create($container, $this);
+
+        $this->setContainer($container);
+        $this->setBuilder($builder);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function _before(TestInterface $test)
+    {
+        $this->taskEnvUp()
+            ->run()
+            ->stopOnFail();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function _after(TestInterface $test)
+    {
+        $this->taskEnvDown()
+            ->run()
+            ->stopOnFail();
+    }
+
+    /**
+     * Clones magento cloud template from git
+     *
+     * @param string|null $version
+     * @return bool
+     * @throws \Robo\Exception\TaskException
+     */
+    public function cloneTemplate(string $version = null): bool
+    {
+        $gitTask = $this->taskGitStack()
+            ->exec('git init')
+            ->exec('git remote -vvv')
```

Comment:
why do you need to run `git remote -vvv`?  Looks like this command is redundant.

Reply:
Yes, it should be removed.
I used it during development.
Thanks!